contracts/standard/arbitration/ArbitrableDeposit.sol

```diff
@@ -37,9 +37,9 @@ contract ArbitrableDeposit is Arbitrable {
     uint8 constant CLAIMANT_WINS = 2;
     string constant RULING_OPTIONS = "Owner wins;Claimant wins"; // A plain English of what rulings do. Need to be redefined by the child class.
 
-    modifier onlyOwner{ require(msg.sender == address(owner)); _; }
-    modifier onlyNotOwner{ require(msg.sender != address(owner)); _;}
-    modifier onlyClaimant{ require(msg.sender == address(claimant)); _;}
+    modifier onlyOwner{require(msg.sender == address(owner), "Can only be called by the owner."); _;}
+    modifier onlyNotOwner{require(msg.sender != address(owner), "Cannot be called by the owner."); _;}
+    modifier onlyClaimant{require(msg.sender == address(claimant), "Can only be called by the claimant."); _;}
```

Comment:
The doc uses space before and after for one line modifier. So let's follow the doc.

Reply:
Solium doesn't allow changing this. Don't you think it's OK as long as it is consistent?